app/controllers/api/steps_controller.rb

```diff
@@ -0,0 +1,32 @@
+module Api
+  class StepsController < Api::AbstractController
+
+    before_action :must_own_sequence
+
+    def create
+      mutate Steps::Create.run(params, sequence: sequence)
+    end
+
+    def destroy
+      if step && step.destroy
+        render nothing: true
+      end
+    end
+
+  private
+
+    def sequence
+      @sequence ||= Sequence.find(params[:sequence_id])
+    end
+
+    def step
+      @step ||= Step.find(params[:id])
+    end
+
+    def must_own_sequence
+      if sequence.user != current_user
+        raise Errors::Forbidden, "Not your Sequence object."
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.